sdk/nodejs/yaml/yaml.ts

```diff
@@ -590,72 +590,72 @@ import * as outputs from "../types/output";
          * For example:
          *     getResourceProperty("v1/Service", "nginx", "spec")
          */
-        public getResourceProperty(groupVersionKind: "admissionregistration.k8s.io/v1/MutatingWebhookConfiguration", name: string, property: "api_version"): pulumi.Output<"admissionregistration.k8s.io/v1">;
-        public getResourceProperty(groupVersionKind: "admissionregistration.k8s.io/v1/MutatingWebhookConfiguration", namespace: string, name: string, property: "api_version"): pulumi.Output<"admissionregistration.k8s.io/v1">;
+        public getResourceProperty(groupVersionKind: "admissionregistration.k8s.io/v1/MutatingWebhookConfiguration", name: string, property: "apiVersion"): pulumi.Output<"admissionregistration.k8s.io/v1">;
```

Comment:
Was this supposed to change? I think the change looks correct, so I'm guessing this was a bug you uncovered while updating the generator code?

Reply:
Yes - I believe this was a bug previously that was fixed incidentally as part of other fixes in `typegen.go`.